server/api/main.py

```diff
@@ -184,6 +185,8 @@ async def setup_api():
     initialize_logs_dir()
     initialize_db()
 
+    server.api.utils.runtimes.nuclio.resolve_nuclio_version()
```

Comment:
can you explain why you need it here?
this method is called in the front end spec and client spec which means when a client tries to connect. shouldn't it be enough or you have to have it at start up?

Reply:
Ok, following the last set of changes, it's not necessary anymore in order to resolve [ML-7284](https://iguazio.atlassian.net/browse/ML-7284). I removed it based on that together with your assessment that it should be covered by the other calls.